pkg/models/cedar_authority_to_operate.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"github.com/guregu/null/zero"
+)
+
+// CedarAuthorityToOperate is the model for ATO information that comes back from the CEDAR Core API
+type CedarAuthorityToOperate struct {
```

Comment:
Looking at the CEDAR Swagger, almost all of these fields are optional in CEDAR except for `CedarID`. I'm wondering if we should reflect that they're optional in this struct by using `zero.Bool`, `zero.String`, etc., similar to our structs for `CedarDeployment` and `CedarRole`. This would also entail using the `zero.BoolFrom()`/`zero.StringFrom()`/etc. functions when parsing the CEDAR response to our model in the CEDAR client, and using the `.Ptr()` methods in our GraphQL resolvers. Thoughts?

Reply:
I can definitely make that change - shouldnt take too long

Reply:
Changes made.